purchases/src/main/kotlin/com/revenuecat/purchases/models/GoogleStoreProduct.kt

```diff
@@ -154,6 +155,18 @@ data class GoogleStoreProduct(
             offeringId,
         )
     }
+
+    /**
+     * Null for INAPP products. The price of the [StoreProduct] in the given locale in a monthly recurrence.
+     * This means that, for example, if the period is annual, the price will be divided by 12.
+     * It uses a currency formatter to format the price in the given locale.
+     * Note that this value may be an approximation.
+     * This value will use the basePlan to calculate the value.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    override fun formattedPricePerMonth(locale: Locale): String? {
+        return subscriptionOptions?.basePlan?.pricingPhases?.last()?.formattedPriceInMonths(locale)
```

Comment:
Note that I'm using the base plan in google, which is what we use to get the `price` field as well so it makes the most sense.